src/main/java/com/backbase/billpay/fiserv/payees/PaymentServicesMapper.java

```diff
@@ -24,18 +26,22 @@
     public List<PaymentService> toPaymentServices(PayeeSummary source) {
         ArrayList<PaymentService> paymentServices = new ArrayList<>();
         PaymentService regularPaymentService = new PaymentService()
-                        .withCutoffTime(source.getCutoffTime())
-                        .withEarliestPaymentDate(fromFiservDate(source.getEarliestPaymentDate()))
-                        .withNextPaymentDate(fromFiservDate(source.getNextPaymentDate()))
+                        .withCutoffTime(toZonedDateTime(source.getCutoffTime()))
+                        .withEarliestPaymentDate(toZonedDateTime(fromFiservDate(
```

Comment:
This is quite confusing to read and makes lots of unnecessary conversions. The Fiserv date comes back as a BldrDate with just contains a string, you only need to use LocalDate.parse(dateString) to convert it to a LocalDate. This would be best as a common method in FiservUtils.